bundles/binding/org.openhab.binding.rwesmarthome/src/main/java/org/openhab/binding/rwesmarthome/internal/communicator/exceptions/LogoutNotificationException.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rwesmarthome.internal.communicator.exceptions;
+
+/**
+ * @author ollie-dev
+ *
+ */
+public class LogoutNotificationException extends Exception {
```

Comment:
The polling of updates normally works for hours, but after about six hours, a logoutNotification occurs, which in my opinion can be handled best with an exception.